src/pybind/mgr/dashboard/frontend/src/index.html

```diff
@@ -3,10 +3,9 @@
 <head>
   <meta charset="utf-8">
   <title>Ceph</title>
-  <base href="/">
 
   <script>
-    window['base-href'] = window.location.pathname;
+    document.URL = '<base href="' + document.location+ '" />';
```

Comment:
Are you sure this works and it's portable across browsers? [Here](https://developer.mozilla.org/en-US/docs/Web/API/Document/URL) it says `document.URL` is read-only? 🤔

Reply:
Yes, but it worked for me in both Firefox and Chrome. I also agree that it's not considered a good practice to modify `document.whatever`. Looking for other options (I'm just too bad at this, just doing some trial & error here and there!)